detector.go

```diff
@@ -119,6 +119,33 @@ func (c *Client) GetDetector(ctx context.Context, id string) (*detector.Detector
 	return finalDetector, err
 }
 
+// GetDetectors gets all detectors.
+func (c *Client) GetDetectors(ctx context.Context, limit int, name string, offset int) ([]*detector.Detector, error) {
+	params := url.Values{}
+	params.Add("limit", strconv.Itoa(limit))
+	params.Add("name", name)
+	params.Add("offset", strconv.Itoa(offset))
```

Comment:
any validation on those parameters?

Reply:
I didn't add any validation because I imitated the same style found in other "get x objects" functions (specifically the `GetIncidents` function), and I chose to do the same to remain consistent. However, I will add it if you think it's important.

Reply:
should be ok if it's consistent, thanks for explaining